docs/plugin-synopsis.asciidoc.erb

```diff
@@ -39,6 +39,15 @@ Available configuration options:
    else
      annotation += "|No"
    end
+   if section == "filter" || section == "output"
+     if config[:allow_dynamic] == true
+       annotation += "|Yes"
+     elsif config[:allow_dynamic] == false
+       annotation += "|No"
+     else
+       annotation += "|Unknown"
```

Comment:
`nil` is used as third possibility forcing the config to be explicitly true of false. maybe 'undocumented' is better than 'unknown' but the result is the same